phpseclib/Crypt/Random.php

```diff
@@ -147,15 +147,17 @@ function crypt_random_string($length)
             session_cache_limiter('');
             session_start();
 
-            $v = $seed = $_SESSION['seed'] = pack('H*', sha1(
-                serialize($_SERVER) .
-                serialize($_POST) .
-                serialize($_GET) .
-                serialize($_COOKIE) .
-                serialize($GLOBALS) .
-                serialize($_SESSION) .
-                serialize($_OLD_SESSION)
-            ));
+            if (function_exists('json_encode')) {
```

Comment:
Is this check (still) necessary? It seems that `json_encode()` is not used (any longer).